filebeat/input/log/harvester.go

```diff
@@ -461,10 +461,7 @@ func (h *Harvester) onMessage(
 				"_id": id,
 			}
 		}
-	} else if &text != nil {
```

Comment:
`text` is initialized as `text := string(message.Content)` around line 424. In Go a variable of type `string` is always a 2-word structure. The first word holds a pointer to the value and the second word embed the length of the value.
So unlike in other programming languages like C in Go a variable of type `string` can never be `nil`.